pkg/storage/trb_request_form.go

```diff
@@ -0,0 +1,162 @@
+package storage
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// CreateTRBRequestForm creates a new TRB request form record in the database
+func (s *Store) CreateTRBRequestForm(ctx context.Context, form *models.TRBRequestForm) (*models.TRBRequestForm, error) {
+	form.ID = uuid.New()
```

Comment:
Just thoughts here, I don't think it needs/ should change. I've seen in other places checking if there is a value for the form.ID, and if not get a new UUID. That would allow you to set explicit uuid from the store method (useful for the current seeding paradigm, but I think eventually that could be changed). 

The GQL resolver will never provide an ID, so for real use case, there shouldn't be an id provided. (So this observation doesn't matter)